src/app.rs

```diff
@@ -395,8 +397,10 @@ impl Application {
                                 } else {
                                     emit!(VectorConfigLoadError);
                                 }
-                            }
-                            _ => break signal,
+                            },
+                            Err(RecvError::Lagged(amt)) => warn!("Overflow, dropped {} signals", amt),
```

Comment:
```suggestion
                            Err(RecvError::Lagged(amt)) => warn!("Overflow, dropped {} signals.", amt),
```

I think this is what's failing the `check-events` part of the build.